src/clarity-angular/datagrid/datagrid-column.spec.ts

```diff
@@ -79,46 +81,77 @@ export default function(): void {
             });
 
-            it("receives an input for the property name", function() {
-                this.context = this.create(DatagridColumn, SimpleTest, [Sort, Filters]);
+            it("receives an input for the property name", function () {
+                this.context = this.create(DatagridColumn, SimpleTest, [Sort, FiltersProvider]);
                 this.context.testComponent.field = "test";
                 this.context.detectChanges();
                 expect(this.context.clarityDirective.field).toBe("test");
             });
 
-            it("accepts a custom filter in the projected content", function() {
-                this.context = this.create(DatagridColumn, FilterTest, [Sort, Filters]);
-                expect(TestBed.get(Filters).getActiveFilters()).toEqual([this.context.testComponent.filter]);
+            it("receives an input for the property filter value", function () {
+                this.context = this.create(DatagridColumn, PreFilterTest, [Sort, FiltersProvider]);
+                this.context.testComponent.field = "test";
+                this.context.testComponent.filterValue = "M";
+                this.context.detectChanges();
+                expect(this.context.clarityDirective.filterValue).toBe("M");
+            });
+
+            it("offers two-way binding on the sorted state", function () {
+                this.context = this.create(DatagridColumn, SimpleTest, [Sort, FiltersProvider]);
+                this.comparator = new TestComparator();
+                this.context.testComponent.comparator = this.comparator;
+                this.context.testComponent.sorted = true;
+                this.context.detectChanges();
+                expect(this.context.clarityDirective.sorted).toBe(true); //dg col instance
+                this.context.getClarityProvider(Sort).clear();
+                this.context.detectChanges();
+                expect(this.context.testComponent.sorted).toBe(false);
             });
 
-            it("accepts a custom string filter in the projected content", function() {
-                this.context = this.create(DatagridColumn, StringFilterTest, [Sort, Filters]);
-                this.stringFilter = this.context.testComponent.stringFilter;
-                // We make the filter active to see if the Filters provider knows about it
+            it("offers two way binding on the filtered state", function () {
+                this.context = this.create(DatagridColumn, PreFilterTest, [Sort, FiltersProvider]);
+                this.context.testComponent.field = "test";
+                this.context.testComponent.filterValue = "M";
+                this.context.clarityDirective.filterValueChange.subscribe((m: string) => {
+                    expect(m).toBe("M");
+                });
+                this.context.detectChanges();
+                expect(this.context.clarityDirective.filterValue).toBe("M");
```

Comment:
Same here, that's not how you want to test 2-way binding. Your test component already uses two-way binding, that's what you want to test:

```typescript
this.context.testComponent.filterValue = "M";
this.context.detectChanges();
expect(this.context.clarityDirective.filterValue).toBe("M");
this.context.clarityDirective.filterValue = "t";
this.context.detectChanges();
expect(this.context.testComponent.filterValue).toBe("t");
```